test/validation/api/time/time.c

```diff
@@ -589,6 +680,50 @@ static void time_test_accuracy_nsec(void)
 	}
 }
 
+static int time_test_global_sync_thr(void *arg ODP_UNUSED)
+{
+	global_shared_mem_t *global_mem;
+	odp_shm_t global_shm;
+	int tid = odp_thread_id();
+
+	global_shm = odp_shm_lookup(GLOBAL_SHM_NAME);
+	global_mem = odp_shm_addr(global_shm);
+
+	if (!global_mem)
+		return 1;
+
+	odp_barrier_wait(&global_mem->test_barrier);
+	global_mem->time[tid] = odp_time_global();
```

Comment:
To better detect if time is running at different pace, a second time could be recorded after a small delay.
E.g.
```
odp_time_wait_ns(ODP_TIME_MSEC_IN_NS * 100);
odp_barrier_wait(&global_mem->test_barrier);
global_mem->time2[tid] = odp_time_global();
```

Reply:
v2.